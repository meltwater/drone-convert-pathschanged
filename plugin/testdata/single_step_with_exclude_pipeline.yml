---
kind: pipeline
type: docker
name: default

steps:
- name: message
  image: busybox
  commands:
  - echo "README.md was not changed"
  when:
    paths:
      exclude:
      - README.md
