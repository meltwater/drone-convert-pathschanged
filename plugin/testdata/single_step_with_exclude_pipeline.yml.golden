kind: pipeline
type: docker
steps:
- when:
    paths:
      exclude:
      - README.md
    event:
      exclude:
      - '*'
  commands:
  - echo "README.md was not changed"
  image: busybox
  name: message
name: default
